spec/rails_helper.rb

```diff
@@ -12,13 +12,21 @@
 require "capybara/rspec"
 require "capybara/rails"
 require "database_cleaner"
+require "capybara/poltergeist"
 
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 ActiveRecord::Migration.maintain_test_schema!
 
-Capybara.register_driver :selenium do |app|
-  Capybara::Selenium::Driver.new(app, browser: :chrome)
+Capybara.register_driver :poltergeist_debug do |app|
+  Capybara::Poltergeist::Driver.new(app, js_errors: false, inspector: true)
+end
+
+Capybara.javascript_driver = :poltergeist_debug
+
+Capybara.server do |app, port|
+  require 'rack/handler/puma'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.